modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -531,6 +539,18 @@ private ObjectDefinition _addObjectDefinition(
 		return _objectDefinition;
 	}
 
+	private void _validateGetObjectDefinitionsPageWithFilter(
```

Comment:
IMHO, this method should be called `_assertGetObjectDefinitionsPageWithFilter`. What do you think?

Reply:
Agreed